### ISSUE TITLE
[Accessibility]A11y__AzureDataStudio_CreateProxy _K4D: Name and Control type property of the “Credential Name” Combo box are incorrect

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5618]
[Check out Keros](
https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
 GitHubTags: #A11yMAS; #A11yTCS; #MAS4.2.1; #A11y_AzureDataStudio; #Win32; #DesktopApp
**Environment Details:**
Application name: Azure Data Studio (Agent, Create External Data, Notebooks)
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763

**Additional Details:**
Tool Name:K4D
MAS Reference: MAS4.1.2,MAS4.2.1

**Repro Steps:**
1.Launch the Azure Data Studio application.
2.Enable the SQL server agent extension.
3.Naviagte to the connected server->Select the database->Navigate to the SQL agent tab->Create Proxy tab.
4. Open K4D tool.
5.Hover over the “Credential Name” Combo box.

**Actual:**
Incorrect  name and control type for “Credential Name” Combo box by using K4D tool .

**Same issue observed for the below page:**

**1.CreateConntionDataSrc:** Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button.

**Expected:**
“Credential Name” Combo box should have control type as “Combo box” and name as Credential Name”.

**User Impact:**
User will not be able to correctly identify the purpose of the control type.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
**MAS4.1.2 -**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**MAS4.2.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}
Attachment for Reference:
![image](https://user-images.githubusercontent.com/39204909/58317675-eb711780-7e33-11e9-83ba-dd7275f85e4f.png)

Does this issue occur when all extensions are disabled?: Yes
